lib/msal-browser/src/cache/TokenCache.ts

```diff
@@ -41,385 +40,385 @@ export type LoadTokenOptions = {
 };
 
 /**
- * Token cache manager
+ * API to load tokens to msal-browser cache.
+ * @param request
+ * @param response
+ * @param options
+ * @returns `AuthenticationResult` for the response that was loaded.
  */
-export class TokenCache implements ITokenCache {
-    // Flag to indicate if in browser environment
-    public isBrowserEnvironment: boolean;
-    // Input configuration by developer/user
-    protected config: BrowserConfiguration;
-    // Browser cache storage
-    private storage: BrowserCacheManager;
-    // Logger
-    private logger: Logger;
-    // Crypto class
-    private cryptoObj: ICrypto;
-
-    constructor(
-        configuration: BrowserConfiguration,
-        storage: BrowserCacheManager,
-        logger: Logger,
-        cryptoObj: ICrypto
-    ) {
-        this.isBrowserEnvironment = typeof window !== "undefined";
-        this.config = configuration;
-        this.storage = storage;
-        this.logger = logger;
-        this.cryptoObj = cryptoObj;
+export async function loadExternalTokens(
+    request: SilentRequest,
+    response: ExternalTokenResponse,
+    options: LoadTokenOptions,
+    cache: TokenCache
```

Comment:
You can take in a `Configuration` object here, the same one accepted on PCA, to remove the dependency on `TokenCache`

Reply:
Done!